index.js

```diff
@@ -30,6 +30,10 @@ function setPropertyForTypes(types, property, value) {
     this.serverless.service.provider.compiledCloudFormationTemplate;
   Object.keys(template.Resources).forEach(function (key) {
     if (types.includes(template.Resources[key]["Type"])) {
+      template.Resources[key]["Properties"] =
```

Comment:
Is null is an acceptable value?

if it must be an object
typeof template.Resources[key]["Properties"] === 'object' : template.Resources[key]["Properties"] ? {}

This is also rather inefficient.  Cashing look ups for multiple hits is advisable, so like template.Resources[key] is hit a bunch of different ways, 
var resources = template.Resources[key] under the keys loop...not needed but it is considered better practice.